panko/pako/src/App.tsx

```diff
@@ -27,10 +32,11 @@ export default function App() {
           setIndex(arr.length - 1);
           return;
         }
-      } catch { }
-    } else {
-      fetchRandom();
+      } catch {
+        // fall through
```

Comment:
something seems a little funky on startup. We want it to jump right in, I think. Maybe we could do something fancier ...

Reply:
TODO: bring home Korean<->English app